R/module_transform_data.R

```diff
@@ -44,37 +45,34 @@ ui_transform_data <- function(id, transforms, class = "well") {
         ),
         div(
           id = wrapper_id,
-          ui_teal_data(id = ns(name), data_module = transforms[[name]]$ui)
+          ui_teal_data(id = ns(name), data_module = transformers[[name]]$ui)
         )
       )
     }
   )
 }
 
 #' @rdname module_transform_data
-srv_transform_data <- function(id, data, transforms, modules, is_transformer_failed = reactiveValues()) {
+srv_transform_data <- function(id, data, transformers = list(), modules, is_transformer_failed = reactiveValues()) {
   checkmate::assert_string(id)
   assert_reactive(data)
-  checkmate::assert_list(transforms, "teal_transform_module", null.ok = TRUE)
+  checkmate::assert_list(transformers, "teal_transform_module")
   checkmate::assert_class(modules, "teal_module")
-  if (length(transforms) == 0L) {
```

Comment:
This wasn't necessary as 

```r
    Reduce(
      function(previous_result, name) {
        stop("I WOULD FAIL IF X HAS A LENGTH > 0")
      },
      x = character(0),
      init = 1
    )
 #> [1]  1
```